tests/visual/src/misc/colorpicker-rtl.html

```diff
@@ -60,7 +60,7 @@
                     </div>
                 </div>
             </section>
-            <section class="k-rtl">
```

Comment:
Is this the fix for the ColorPicker that we sync before? If yes, then please cherry-pick it to the `fix-color-gradient-rtl` branch.